### PR TITLE
feat(JSG-236): export and review deploy files

### PR DESCRIPTION
**Pull request by Emmeggi95**
## Description

Added section, in the gitops-based deploy page, about the export and review functionality.

## Pull Request Type

- [x] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [x] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [x] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [x] No sensible content has been committed

## Other information

TODO: replace screenshots when the frontend work is done.

### CODE REVIEW

File: docs/development_suite/deploy/gitops-based/index.md [hunk 1]
```diff
@@ -34,6 +34,16 @@ Another benefit is that **the cluster does not need to be accessed directly**, r
 
 Moreover, in push-based deployment models, pipelines often require access to credentials to authenticate with the cluster and make changes. Storing credentials within pipelines increases the risk of inadvertent exposure or misuse. In GitOps, **credentials are typically configured within the GitOps tool** which often resides on the cluster itself and has specific authorization rules that are not shared with the pipeline code and context.
 
+## Export and review configuration files
+
+When using **Enhanced Project Workflow**, Kubernetes files are generated on deployment. After selecting the desired revision, you will have the possibility to export and download the files in `tag.gz` format, or to review them directly inside the browser.
```
Comment (davidebianchi) [inline docs/development_suite/deploy/gitops-based/index.md:39]: It is possible to select also the version, not only the revision. Is it correct?

Comment (Emmeggi95) [reply docs/development_suite/deploy/gitops-based/index.md:39]: Yes, how should I refer to it?

Comment (davidebianchi) [reply docs/development_suite/deploy/gitops-based/index.md:39]: We use somewhere the `reference` keyword?
We could:

1. After selecting the desired reference (revision or version) ...
2. After selecting the desired revision (or version), you will ...

What do you think about?

Comment (Emmeggi95) [reply docs/development_suite/deploy/gitops-based/index.md:39]: I prefer the second option, I don't find other `reference` usages with this meanings and it sounds to me like a dev term

**Review by fredmaggiowski — APPROVED** [review]
LGTM imho we can merge it and then update the screenshot when we have improved the UI

**Review by fredmaggiowski — APPROVED** [review]

**Pull request merged**